x-pack/plugins/cloud_security_posture/server/routes/compliance_dashboard/compliance_dashboard.ts

```diff
@@ -100,19 +45,20 @@ export const defineGetComplianceDashboardRoute = (
     async (context, _, response) => {
       try {
         const esClient = context.core.elasticsearch.client.asCurrentUser;
-        const latestCyclesIds = await getLatestCyclesIds(esClient);
+
+        const { id: pitId } = await esClient.openPointInTime({
+          index: LATEST_FINDINGS_INDEX_PATTERN,
+          keep_alive: '30s',
+        });
+
         const query: QueryDslQueryContainer = {
-          bool: {
-            should: latestCyclesIds.map((id) => ({
-              match: { 'cycle_id.keyword': { query: id } },
-            })),
-          },
+          match_all: {},
         };
 
         const [stats, resourcesTypes, clustersWithoutTrends, trends] = await Promise.all([
-          getStats(esClient, query),
-          getResourcesTypes(esClient, query),
-          getClusters(esClient, query),
+          getStats(esClient, query, pitId),
+          getResourcesTypes(esClient, query, pitId),
+          getClusters(esClient, query, pitId),
```

Comment:
no need for PIT for trends?

Reply:
No, it queries a different index (the score index)